docs: add docs for vulnerability related commands (#3362)
canonical/ubuntu-pro-client: docs/howtoguides/how_to_use_manifest_file_for_pro_vulnerability.rst

How to provide a manifest file to pro vulnerability commands
************************************************************

> If you want to provide a manifest file to either ``pro vulnerability show`` or

Reviewer: Why would someone want to do that?

Author: You might want the see the CVE status for a different machine than the one you are using. The manifest file is the way to achieve that

Reviewer: Can we mention that here as an example use case? (I'm not sure it's that obvious)

``pro vulnerability list``, you can use the flag ``--manifest-file``. For example:

.. code-block:: bash

   pro vulnerability list --manifest-file=MANIFEST_FILE_PATH

> Now, be aware that when providing a manifest file to either of the commands, we will
> interpret that the manifest file was generated for the **same** ubuntu release the
> the machine is running on. For example, if you are running on the command on Xenial
> machine and provides a manifest for the command, the command will believe that the
> manifest also represents a Xenial machine. If that is not the case, you **need** to
> provide the ``--series`` parameter to the command as well, to provide the command
> with the ubuntu release the manifest was generated on, for example:

Reviewer: ```suggestion
When providing a manifest file to either of the commands, it is assumed
that the manifest file was generated for the **same** Ubuntu series the
the machine is running on. For example, if you run the command on a Xenial
machine, and provide a manifest file for the command, the command will assume that the
manifest also represents a Xenial machine. If that is not the case, you must
provide the ``--series`` parameter to the command as well, to provide the Ubuntu series the manifest was generated on, for example:
```

.. code-block:: bash

   pro vulnerability list --manifest-file=MANIFEST_FILE_PATH --series=focal

> Currently, we only support manifest files generated by the Pro client. This

Reviewer: ```suggestion
Currently, we only support manifest files generated by the Pro Client. This
```

> Finally, today we only support the manifest file generated by the Pro client. This

Reviewer: ```suggestion
Currently, we only support manifest files generated by the Pro client. This
```
